packages/core/.npmrc

```diff
@@ -0,0 +1,2 @@
+engine-strict=true
+resolution-mode=highest
```

Comment:
Haven't messed with the `resolution-mode` setting, was this something you set? What's reasoning? (Might be worth a comment, especially if we want to set this as a default across other projects)